tests/bluechi_test/bluechictl.py

```diff
@@ -149,6 +149,36 @@ def reset_failed(
             expected_result,
         )
 
+    def get_default_target(
+        self,
+        node_name: str = "",
+        check_result: bool = True,
+        expected_result: int = 0,
+    ) -> Tuple[Optional[int], Union[Iterator[bytes], Any, Tuple[bytes, bytes]]]:
+        cmd = f"get-default {node_name}".strip()
+        return self._run(
+            f"GetDefaultTarget of node {node_name}",
+            cmd,
+            check_result,
+            expected_result,
+        )
+
+    def set_default_target(
+        self,
+        node_name: str = "",
+        target: str = "",
+        force: str = "false",
+        check_result: bool = True,
+        expected_result: int = 0,
+    ) -> Tuple[Optional[int], Union[Iterator[bytes], Any, Tuple[bytes, bytes]]]:
+        cmd = f"set-default {node_name} {target} {force}".strip()
```

Comment:
Same as above, is `strip()` really needed?

Reply:
Changed